extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -329,8 +330,9 @@ export async function getCompletionItemsFromSpecs(specs: Fig.Spec[], terminalCon
 
 	if (!specificSuggestionsProvided && (filesRequested === foldersRequested)) {
 		// Include builitin/available commands in the results
+		const labels = new Set(items.map(i => i.label));
```

Comment:
We might want an approach that actually merges them eventually here, the ones we provide here might end up being richer. For example the pwsh ones I was working on will have descriptions as provided by pwsh.